src/locales/pl.ts

```diff
@@ -90,34 +90,34 @@ export default {
     peter_and_paul_apostles: 'Świętych Apostołów Piotra i Pawła',
     presentation_of_the_lord: 'Ofiarowanie Pańskie',
     transfiguration: 'Przemienienie Pańskie',
-    trinity_sunday: 'Niedziela Najświętszej Trójcy',
+    trinity_sunday: 'Najświętszej Trójcy',
```

Comment:
As we try to use the celebration names from the official translation of Roman Missal in the particular vernacular, I think this should be changed to [`Trójcy Przenajświętszej`](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf#page=83).

Also note that I haven’d proofed any other celebration names. @jarosz, could you please proof the celebration names from the Polish Roman Missal? Ideally from _editio typica tertia_, however, if you can’t access it (or it is not published yet in Polish), you could use _editio typica altera_ or even the above-linked [1986 _editio  typica_](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf).